projects/aca-shared/testing/pages/login-page.ts

```diff
@@ -44,14 +44,17 @@ export class LoginPage extends Page {
   async loginWith(username: string, password?: string) {
```

Comment:
we don't need another login page rather than the one in adf-testing

Reply:
The goal of this PR is to move the testing framework to a reusable project, so DW team can use it.
Refactoring ACA framework was not considered as part of this ticket.